.golangci.yml

```diff
@@ -158,54 +140,67 @@ issues:
     #
     # The default list of exclusions can be found at:
     # https://golangci-lint.run/usage/false-positives/#default-exclusions
+    generated: lax
```

Comment:
Not 100% sure if it found any generated files, otherwise we could remove this, and use its default (`strict`)

Reply:
☝️ just gave it a try, and I _think_ it's happy with this removed (using the default)